tests/helm/test_runner.py

```diff
@@ -62,6 +62,24 @@ def test_runner_honors_enforcement_rules(self):
         self.assertEqual(len(report.passed_checks), 0)
         self.assertEqual(len(report.skipped_checks), 0)
         self.assertEqual(len(report.parsing_errors), 0)
+        
+    @unittest.skipIf(not helm_exists(), "helm not installed")
+    def test_runner_invalid_chart(self):
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+        scan_dir_path = os.path.join(current_dir, "runner", "resources", "schema-registry")
+
+        runner = Runner()
+        filter = RunnerFilter(framework=['helm'], use_enforcement_rules=False)
+        # this is not quite a true test, because the checks don't have severities. However, this shows that the check registry
+        # passes the report type properly to RunnerFilter.should_run_check, and we have tests for that method
+        try:
+            report = runner.run(
+                root_folder=scan_dir_path, runner_filter=filter
+            )
+        except Exception:
+            assert 1 == 2
```

Comment:
```suggestion
        report = runner.run(
            root_folder=scan_dir_path, runner_filter=filter
        )
```
the test will fail if an exception is raised